learn/educators/python.md

```diff
@@ -0,0 +1,98 @@
+"editor.suggest.filterGraceful": true,
+"editor.suggestOnTriggerCharacters": false,
+"editor.acceptSuggestionOnEnter": "on",
+"editor.suggest.showIcons": false,
+"editor.suggest.maxVisibleSuggestions": 7,
+"editor.hover.enabled": false,
+"editor.hover.sticky": false,
+"editor.suggest.snippetsPreventQuickSuggestions": false,
+"editor.parameterHints.enabled": false,
+"editor.wordBasedSuggestions": true,
+"editor.tabCompletion": "on",
+"extensions.ignoreRecommendations": true,
+"files.autoSave": "afterDelay",
+```
+
+
+Finally, he also utilizes the built-in terminal heavily in his class as an introduction to running programs from the command line and navigating around their machine. He appreciates how "the built-in terminal panel does not try to automate too much (which, if it did, would deprive newcomers of the experience of the information-flow that's going on)."
+
+Thank you, Professor Dodds, for sharing your story! If you’re interested in using Visual Studio Code to teach Python in your classes, you can get started with the Python Education Extension Pack above!
+
+## Coding Pack for Python (Windows)
+
+This coding pack includes everything you need to start coding in Python:
+- VS Code
+- Python and LiveShare extensions in VS Code
+- Python 3.8.5 and add it to the PATH
+- Python packages including, jupyter, numpy, sklearn, pandas, and matplotlib
+
+Full details and a link to install is available under [Coding Packs](/installers#_coding-pack-for-python).
+
+## Microsoft Learn free Python programming and data science lessons
+
+### NASA-Inspired Lessons
+This learning path enables students to use Python to explore doing analyses and projects inspired from real-world problems faced by National Aeronautics and Space Administration (NASA) scientists. View full details of the lessons under [NASA-Inspired Lessons](https://code.visualstudio.com/learn/students/nasa-python)
+
+![NASA Inspired Python lessons](images/python/nasa-learning-path.png)
+
+### Learn Python with Over The Moon
+These space-themed lessons were inspired by the Netflix film, [Over the Moon](https://www.youtube.com/watch?v=26DIABx44Tw), and will introduce students to data science, machine learning, and artificial intelligence using Python and Azure. View full details on [Learn Python with Over The Moon](https://code.visualstudio.com/learn/students/over-the-moon-python).
+
+![Learn Python with Over The Moon](images/python/over-the-moon-learning-path.png)
+
+### Wonder Woman Inspired Lessons
+Give an introduction to Python with WONDER WOMAN 1984 inspired lessons that help students learn about the basics like conditionals and variables. Get full lesson details under [Learn Python with Wonder Woman](https://code.visualstudio.com/learn/students/wonder-woman-python)
+
+![Wonder Woman Inspired Python Lessons](images/python/wonder-woman-learning-path.png)
+
+### [Write basic Python in Notebooks](https://docs.microsoft.com/learn/modules/basic-python-nasa/)
```

Comment:
You may want to be consistent with the H2s. Since some aren't links, I'd leave the rest as plan text and have links below